adoc/bp_chap_suma_backup.adoc

```diff
@@ -163,58 +166,41 @@ Stopping the SUSE Manager database...
+You can check your current database size by navigating to [path]``/var/lib/pgsql/`` and running [command]``df -h``.
 
+The [command]``smdba`` tool also manages your archives, keeping only the most recent backup, and the current archive of logs.
+The log files can only be a maximum file size of 16{nbsp}MB, so a new log file will be created once the files reach this size.
+Every time you create a new backup, previous backups will be purged to release disk space.
+We recommend you use [command]``cron`` to schedule your [command]``smdba`` backups to ensure that your storage is managed effectively, and you always have a backup ready in case of failure.
 
-A cold, or offline, database backup occurs when the service is completely shut down at the time of the backup.
-This method can reduce the risk of incomplete backups, but does require significant downtime.
-We recommend that you perform a cold backup after initial installation and configuration, and then at weekly or monthly intervals during operation, but rely on hot backups for daily or hourly intervals.
-Additionally, we recommend that you consider backing up your database to an NFS share location, which can itself be backed up.
 
+=== Performing a Manual Database Backup
 
-<!-- TODO: Fixme LKB -->
-When using smdba, you should never require a cold backup.
-You may perform a cold backup once after initial SUSE Manager installation and configuration. {suse}
-recommends creating a full-backup once a month or weekly and an additional incremental backup for single days.
-Moving [path]``/var/spacewalk``
- to an NFS share that is centrally backed-up, will save you alot of time when a restore is required.
-The NFS share may also be used during the migration from SUSE Manager 2.1 to SUSE Manager 3.
+The [command]``smdba`` tool can be run directly from the command line.
+We recommend you run a manual database backup immediately after installation, or if you have made any significant changes to your configuration.
 
-
-<!-- TODO: Fixme LKB -->
-.Procedure: Performing a Cold Backup
-. Step 1
-. Step 2
-. FIXME
-
-
-
-=== Performing a Hot Database Backup
-
-
-A hot, or dynamic, database backup occurs when the service continues normal operation while the backup occurs.
-This method can sometimes result in data loss or conflicts, and can effect database performance while the operation is underway.
-However, hot backups also enable much finer-grained archives, and can often result in seamless restoration from failure, in addition to maintaining uptime.
-We recommend that you perform a hot backup on daily or hourly intervals, to serve as a more complete record between cold backups.
-Additionally, we recommend that you consider backing up your database to an NFS share location, which can itself be backed up.
+[NOTE]
+====
+When [command]``smdba`` is run for the first time, or if you have changed the location of the database, it will need to reboot your database after performing the archive.
```

Comment:
[NOTE]
====
When [command]``smdba`` is run for the first time, or if you have changed the location of the backup, it will need to restart your database before performing the archive.
This will result in a small amount of downtime.
Note that regular database backups will not require any downtime.
====